libs/src/sdk/bootstrapConfig.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * This file is autogenerated by ./scripts/generateBootstrapConfig.ts.
+ * DO NOT EDIT MANUALLY!
+ */
+/* eslint-disable prettier/prettier */
+export const bootstrapConfig = {
```

Comment:
normally i see blockchains specifying separate bootstrap config files for different chains/envs.

i also would prefer that this generated file could be _exactly_ rip & replace with the response from a discovery node health check

wdyt?

Reply:
like imo we should have something more like

src/sdk/config/production.json
src/sdk/config/staging.json

and those files are replaced by CI & the script versus an autogen ts file that way it can be linked out & shared in a very straight forward way

Reply:
I'm down. Initially had that, will go back to it. One thing though, can't have comments in JSON which is annoying. That's why i switched to TS. TS also lets us be sure the keys exist

Reply:
counterpoint, API json can't have comments either tho!

Reply:
I'm not sure I understand? I wanted a comment in the configs so that nobody tries to edit them manually

Reply:
Ah I think I understand? the API returned by DN won't be typed/commented?

Reply:
yeah! that's what i'm thinking. drop in replacement so that the Apis really conform

Reply:
hm i still think types are nice though. If the generated JSON is differently typed, we want to know that and make adjustments. We _could_, however, have the generation be based on the API output, so that _for sure_ the format is the same because otherwise it breaks the type?